src/main/java/wooteco/subway/section/service/SectionService.java

```diff
@@ -0,0 +1,143 @@
+package wooteco.subway.section.service;
+
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import wooteco.subway.exception.InvalidInsertException;
+import wooteco.subway.exception.NotFoundException;
+import wooteco.subway.line.Line;
+import wooteco.subway.section.Section;
+import wooteco.subway.section.Sections;
+import wooteco.subway.section.dto.SectionRequest;
+import wooteco.subway.section.dto.SectionResponse;
+import wooteco.subway.section.repository.SectionDao;
+import wooteco.subway.station.Station;
+import wooteco.subway.station.dto.StationResponse;
+import wooteco.subway.station.service.StationService;
+
+import java.util.List;
+
+@Service
+public class SectionService {
+    private final StationService stationService;
+    private final SectionDao sectionDao;
+
+    public SectionService(StationService stationService, SectionDao sectionDao) {
+        this.stationService = stationService;
+        this.sectionDao = sectionDao;
+    }
+
+    @Transactional
+    public void save(Line newLine, SectionRequest sectionReq) {
+        validateExistStations(sectionReq.getUpStationId(), sectionReq.getDownStationId());
+        sectionDao.save(newLine.getId(), sectionReq.toEntity());
```

Comment:
```suggestion
        sectionDao.save(sectionReq.toEntity(newLine));
```

Section 클래스가 Line을 소유하고 있으니 이렇게 짤 수도 있지 않을까요?
line.getId()를 전달해서 save 한다는 건 db 입장에서 보는거고... section이라는 객체가 완전한 상태로 저장하면 Line을 가지고 있어야 하니 따로 아이디만 전달할 필요는 없어보여요